handlers/feedback_test.go

```diff
@@ -416,12 +320,176 @@ func Test_feedbackThanks(t *testing.T) {
 				},
 			}}
 		Convey("When feedbackThanks is called", func() {
-			feedbackThanks(w, req, url, errorType, mockRenderer, mockNagivationCache, lang)
+			feedbackThanks(w, req, url, mockRenderer, mockNagivationCache, lang)
 			Convey("Then the handler sanitises the request text", func() {
 				dataSentToRender := mockRenderer.BuildPageCalls()[0].PageModel.(model.Feedback)
-				returnToUrl := dataSentToRender.Metadata.Description
+				returnToUrl := dataSentToRender.ReturnTo
 				So(returnToUrl, ShouldEqual, "&lt;script&gt;alert(1)&lt;/script&gt;")
 			})
 		})
 	})
 }
+
+func TestValidateForm(t *testing.T) {
+	Convey("Given the validateForm function", t, func() {
+		testCases := []struct {
+			givenTxt    string
+			given       *model.FeedbackForm
+			expectedTxt string
+			expected    []coreModel.ErrorItem
```

Comment:
I see what you're getting at here - having `expectedTxt` may be create an expectation that *that* text is expected? Maybe there's a better way of describing it.